### PR TITLE
Add SciTokens mappings for ATLAS

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: virtual-organizations/ATLAS.yaml [hunk 1]
```diff
@@ -1,5 +1,19 @@
 AppDescription: ATLAS Production and Analysis
 CertificateOnly: false
+Credentials:
+  TokenIssuers:
+    - Description: production
```
Comment (brianhlin) [inline virtual-organizations/ATLAS.yaml:5]:
```suggestion
    - Description: ATLAS production
```

File: virtual-organizations/ATLAS.yaml [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+      Subject: 7dee38a3-6ab8-4fe2-9e4c-58039c21d817
+      URL: https://atlas-auth.web.cern.ch/
+      DefaultUnixUser: usatlas1
+    - Description: SAM/ETF
```
Comment (brianhlin) [inline virtual-organizations/ATLAS.yaml:9]:
```suggestion
    - Description: ATLAS SAM/ETF
```

File: virtual-organizations/ATLAS.yaml [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+      Subject: 5c5d2a4d-9177-3efa-912f-1b4e5c9fb660
+      URL: https://atlas-auth.web.cern.ch/
+      DefaultUnixUser: usatlas2
+    - Description: analysis
```
Comment (brianhlin) [inline virtual-organizations/ATLAS.yaml:13]:
```suggestion
    - Description: ATLAS analysis
```

**Review by brianhlin — APPROVED** [review]
Minor suggestions for the descriptions since all VO mappings are shoved together in the same generated file. Otherwise LGTM, pre-approving

**Comment by matyasselmeci** [comment]
That's fine although I now add a header between blocks of issuers so it would actually look like
```
## ATLAS ##
# production:
SCITOKEN /.../ usatlas1
# SAM/ETF:
SCITOKEN /.../ usatlas2
# analysis:
SCITOKEN /.../ usatlas3
## GLOW ##
SCITOKEN /.../ glow
```

**Pull request merged**